src/gui/Clipboard.h

```diff
@@ -31,7 +31,8 @@ class Clipboard : public QObject
     Q_OBJECT
 
 public:
-    void setText(const QString& text);
+    void setText(const QString& text, bool secret = false);
```

Comment:
You also need to replace all usages of setText() with the call to the secret version.

Reply:
Why? Non secret field will be visible. Only passwords will be omitted from the clipboard history. 
I've already changed the copy-password to clipboard usages to use the secret version. 
Other fields will call setText with secret set to false by default

Reply:
I don't see any use of the new functions in this PR, except in setClipboardSecretTextAndMinimize() (which is also new).
Besides, I would prefer this flag to be set for everything. Usernames etc. are also cleared from the clipboard after the timeout, so they should also not be in the history.

Reply:
setClipboardSecretTextAndMinimize() itself it's used when copying the password

I will change it since I agree that all field should be treated like secrets

Reply:
If you have any dubts, you may want to ask Martin Flöser, the Klipper developer.
When the patch will be finished, please let me know, so I will test it